systems/llnl-cluster/system.py

```diff
@@ -5,24 +5,35 @@
 
 import pathlib
 
-from benchpark.directives import variant
+from benchpark.directives import variant, system_site, maintainer
 from benchpark.system import System
-
-id_to_resources = {
-    "ruby": {
-        "sys_cores_per_node": 56,
-    },
-    "magma": {
-        "sys_cores_per_node": 96,
-    },
-    "dane": {
-        "sys_cores_per_node": 112,
-    },
-}
+import benchpark.paths
 
 
 class LlnlCluster(System):
 
+    id_to_resources = {
+        "ruby": {
+            "sys_cores_per_node": 56,
+            "hardware_key": str(benchpark.paths.hardware_descriptions)
+            + "/Supermicro-icelake-OmniPath/hardware_description.yaml",
+        },
+        "magma": {
+            "sys_cores_per_node": 96,
+            "hardware_key": str(benchpark.paths.hardware_descriptions)
+            + "/Penguin-icelake-OmniPath/hardware_description.yaml",
+        },
+        "dane": {
+            "sys_cores_per_node": 112,
+            "hardware_key": str(benchpark.paths.hardware_descriptions)
+            + "/DELL-sapphirerapids-OmniPath/hardware_description.yaml",
+        },
+    }
```

Comment:
I refactored this object into the class such that it could be accessed in `info.py` without importing `systems/llnl-cluster/system.py`. I added the new key, `hardware_key` to `id_to_resources`, because this sometimes maps to different yaml's based on the cluster.

1. Should `hardware_key` be a directive instead of existing in `id_to_resources`? 
2. Should `benchpark info system llnl-cluster --hardware` include keys in `id_to_resources`, such as `sys_cores_per_node` or only include the hardware yaml information?